src/parsers/parser.js

```diff
@@ -55,6 +60,11 @@ class Parser {
         else this.throwError(feedbackMessages.genericErrorMsg(this.getCurrentTokenValue()));
     }
 
+    /**
```

Comment:
This comment is reflecting the intent of the previous implementation and it is also not necessary as the method's name describes its function.